### PR TITLE
Redacted Struct

### PR DESCRIPTION
**Pull request by zachmargolis**
A simple `Struct` subclass that redacts members by default when printing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+A version of Ruby's built-in Struct that can mark specific members to be redacted when printing instances:
+
+```ruby
+Config = RedactedStruct.new(
+  :username,
+  :password,
+  :timeout,
+  keyword_init: true,
+  redacted_members: [:password]
```
Comment (zachmargolis) [inline README.md]: Looking for feedback, do we want a shorthand to react all properties?

```
redact_all: true
```

Comment (zachmargolis) [reply README.md]: Updated in 94dc6a4 to default to all redacted, switch to an allowlist

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-stretch-node
```
Comment (mitchellhenke) [inline .circleci/config.yml]: is there a ruby image without node?

**Review by mitchellhenke — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (zachmargolis) [reply .circleci/config.yml]: Yup! Fixed in 9f2046fb7a371bc2b1f100ce5f4262fc3b9ed508

**Pull request merged**